### PR TITLE
Fix 0057 and make add-test.perl easier to use

### PR DESCRIPTION
**Pull request by dbooth-boston**

### CODE REVIEW

File: RDF-Pipeline/t/add-test.perl [hunk 1]
```diff
@@ -9,13 +9,18 @@
...
-where URL is the RDF Pipeline test URL to invoke for the new test.\n";
-my $url = shift @ARGV;
+@ARGV >= 1 or die "Usage: $0 description \n";
+my $url = "http://localhost/node/xxx";
 my $description = join("_", @ARGV) || "";
-$description =~ s/[^a-zA-Z\-\_0-9]//g;
-$description =~ s/\A[^a-zA-Z0-9]*//;
-$description =~ s/[^a-zA-Z0-9]*\Z//;
+# Clean up the description for use as a system-friendly directory
+# name (no weird chars):
```
Comment (glenna-m) [inline RDF-Pipeline/t/add-test.perl:16]: Helpful.  Thank you.

File: RDF-Pipeline/t/add-test.perl [hunk 2]
```diff
@@ -49,7 +54,7 @@
 
 warn "Created test: $dir\n";
 
-# Encourage the user to edit the test description in $readme:
+# Encourage the user to edit the test's $readme and test-script:
```
Comment (glenna-m) [inline RDF-Pipeline/t/add-test.perl:57]: :+1:

File: RDF-Pipeline/t/tests/0057_One_node_pipeline/setup-files/node/myecho [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
 my $qs = $ENV{QUERY_STRING};
-print "$thisuri updater called with $qs\n";
-print "and got $aaResponse\n";
+print "$thisUri updater called with $qs\n";
+print "$thisUri got $aaResponse\n";
+# Change this to 3 to simulate heavy processing:
+my $seconds = 0;
+print "$thisUri sleeping $seconds seconds to simulate heavy processing...\n";
+sleep $seconds if $seconds;
+print "$thisUri Finished.\n";
```
Comment (glenna-m) [inline RDF-Pipeline/t/tests/0057_One_node_pipeline/setup-files/node/myecho:21]: This looks much better.  I picked it up and I am hitting my nodes now executing this version from the VM to nodejs on my Mac.

**Comment by glenna-m** [comment]
It would be good to squash this into one commit instead of 3 before merging it.   Soon as that's done, LGTM - it's good to go.

**Comment by dbooth-boston** [comment]
It turns out that I had intervening merges, so squashing would be too time consuming and painful.  So I'll leave in the extra commits this time, and hopefully next time I'll do it better.

**Pull request merged**